tuneme/templates/polls/detail.html

```diff
@@ -14,6 +14,7 @@
 	<h3>{{ question.title }}</h3>
 	{% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
 		<form action="{% url 'molo.polls:vote' question.id %}" method="post">
+			{% if not user_has_voted %}
```

Comment:
The way I've done it takes away the voting button once the user has voted, and each time after that he is not able to vote as there is not vote button. However, this is only on the poll page. On the home page, the poll with the voting option is there, unless you want me to change the tuneme homepage?